lib/repl.js

```diff
@@ -209,9 +209,9 @@ function REPLServer(prompt,
         // preserve original error for wrapped command
         const error = wrappedErr || e;
         if (isRecoverableError(error, code))
-          err = new Recoverable(error);
+          err = prettifyStackTrace(new Recoverable(error));
```

Comment:
Why not just use `Error.captureStackTrace()`?

Reply:
That's a good question. I guess I was just going with something similar to what @princejwesley did in https://github.com/nodejs/node/pull/9601. But perhaps that's not the best approach. I'll try an impl with `Error.captureStackTrace()` and push it for review.